src/mrtjp/projectred/core/RenderHalo.scala

```diff
@@ -45,7 +45,27 @@ object RenderHalo
         renderList = renderList.sorted
 
         GL11.glPushMatrix()
-        RenderUtils.translateToWorldCoords(Minecraft.getMinecraft.renderViewEntity, event.partialTicks)
+
+        // Removed: To minimize floating point errors we need to minimize calculations (especially 
```

Comment:
If you leave that note in the commit, it will be visible there forever, and won't take up too much space here.